packages/react-reconciler/src/__tests__/ReactSuspenseWithNoopRenderer-test.internal.js

```diff
@@ -488,6 +488,7 @@ describe('ReactSuspenseWithNoopRenderer', () => {
   });
 
   it('tries rendering a lower priority pending update even if a higher priority one suspends', async () => {
+    spyOnDev(console, 'error');
```

Comment:
If you use `spyOnDev` then you should also assert the number and type of warnings at the end of the test. Otherwise this will suppress unexpected warnings.